src/screens/Admin/Instructors/InstructorList.jsx

```diff
@@ -15,7 +15,7 @@ export default function InstructorList({ instructors, loading, currUni, onInacti
   const onSearch = (keyCode) => {

```

Comment:
@angrave if we want, we can make the search happen 'live' with every state change. That way we won't need to even hit enter or click the button now that we're not using `jQuery`

Reply:
This can be a separate commit actually, I'll make this into a new issue so that this PR is fixed on just removing jquery.

Reply:
https://github.com/classtranscribe/FrontEnd/issues/807

Reply:
Sounds good.
@harsh183 
FYI how to properly debounce a search in react-
https://www.alexhughes.dev/blog/settimeout-with-hooks/
https://dev.to/imforja/5-steps-to-perform-a-search-when-user-stops-typing-using-react-hooks-in-a-controlled-component-5edd